po/zh-CN.po

```diff
@@ -17145,7 +17295,7 @@ msgstr ""
 
 #: src/credits.md:1
 msgid "# Credits"
-msgstr ""
+msgstr "# 鸣谢"
```

Comment:
"credits" has translated in #867

Reply:
Strange, I wonder how the translation got copied into here. At least it doesn't conflict :smile: